cloud-annotations/src/main/java/cloud/commandframework/annotations/Flag.java

```diff
@@ -27,11 +27,15 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.util.function.BiFunction;
 
 /**
  * Indicates that the parameter should be treated like a {@link cloud.commandframework.arguments.flags.CommandFlag}.
- * If the parameter is a {@code boolean} then a presence flag will be created, else a value flag will be created
- * and the parser will be resolved the same way as it would for a {@link Argument}
+ * <ul>
+ *     <li>If the parameter is a {@code boolean} then a presence flag will be created</li>
```

Comment:
```suggestion
 *     <li>If the parameter is a {@code boolean}, a presence flag will be created</li>
```
This isn't a programming language; "if then" sounds bad :(